quinn-udp/Cargo.toml

```diff
@@ -14,14 +14,13 @@ workspace = ".."
 all-features = true
 
 [features]
-default = ["log"]
-# Write logs via the `log` crate when no `tracing` subscriber exists
-log = ["tracing/log"]
+default = ["tracing"]
```

Comment:
Note that this pull request proposes dropping the `log` **feature** from `quinn-udp`. A user using `quinn-udp` through `quinn` can enable the `tracing/log` feature through `quinn`'s `log` feature.

Now this does introduce some inconsistency. Alternatively we could:

- Rename the `log` feature to `tracing-log` to disambiguate from the `log` crate. Though I assume you would count this as a breaking change.
- Keep the `log` feature on `quinn-udp`, accepting the confusion that the `log` feature does not enable the `log` crate in `quinn-udp`, but actually forwards to the `tracing/log` feature.

Reply:
Yes, this is definitely a semver-incompatible change for quinn-udp. I think that's probably acceptable, but this PR should contain a commit that bumps the quinn-udp version number to 0.6.0 (and similar for quinn's dependency on quinn-udp).

Because I don't think we want to make a semver-incompatible release for the quinn crate right now, we should take care to make sure that the behavior for quinn is unchanged, but that should give us the freedom to change the quinn-udp interface as we see fit.

Reply:
Per my previous comment, for compatibility with the previous situation we should make sure `default` in quinn-udp enables `tracing/log`. I think we should make that a separate feature `tracing-log = ["tracing", "tracing/log"]`, and change `default` to enable `tracing-log` instead of just `tracing`.

Reply:
That makes sense. Thanks for the suggestion. Can you take a look at https://github.com/quinn-rs/quinn/pull/1921/commits/55f4a8d124a23ff5df881fc8c8b1a03f58d733a6?